packages/fx-core/src/common/spec-parser/utils.ts

```diff
@@ -193,6 +204,27 @@ export function isSupportedApi(
   return false;
 }
 
+export function getAPIKeyAuth(
+  securities: OpenAPIV3.SecurityRequirementObject[] | undefined,
+  spec: OpenAPIV3.Document
+): OpenAPIV3.ApiKeySecurityScheme | undefined {
+  const securitySchemas = spec.components?.securitySchemes;
+  if (securities && securitySchemas) {
+    for (let i = 0; i < securities.length; i++) {
+      const security = securities[i];
+      if (Object.keys(security).length == 1) {
```

Comment:
Based on the method name it is just get the APIKey Auth configuration. So we may not filter here. And how about later we support multiple and logic？ will we change the interface?

Reply:
Thanks, will update the logic to make it more general